modules/claims_api/app/controllers/claims_api/v2/application_controller.rb

```diff
@@ -139,10 +139,12 @@ def build_target_veteran(veteran_id:, loa:) # rubocop:disable Metrics/MethodLeng
         )
         # populate missing veteran attributes with their mpi record
         target_veteran.mpi_record?(user_key: veteran_id)
+
         mpi_profile = target_veteran&.mpi&.mvi_response&.profile || {}
-        target_veteran[:first_name] = mpi_profile[:given_names]&.first
-        if target_veteran[:first_name].nil?
-          raise ::Common::Exceptions::UnprocessableEntity.new(detail: 'Missing first name')
+        if mpi_profile.nil?
```

Comment:
we'll never hit this - we're returning the value or an empty object. That means if it's `nil` originally it will return an empty object, and then if we call `.nil?` on the empty object, we'll get `false`.

```
response = nil || {}
 => {}
2.7.6 :007 > response.nil?
 => false
```